update/delta/delta_1.3.4_005_create_fk_leak_type.sql

```diff
@@ -0,0 +1,40 @@
+ALTER TABLE qwat_od.leak ADD column fk_type integer;
+
+UPDATE qwat_od.leak
+SET fk_type = (
+CASE
+  -- NPS and others from the demo
+WHEN type = 'other' THEN 101
+WHEN type = 'inconnu' THEN 102
+WHEN type = 'to be determined' THEN 103
+WHEN type = 'rupture (break, breakage)' THEN 9201
+WHEN type = 'crack' THEN 9202
+WHEN type = 'hole' THEN 9203
+WHEN type = 'default' THEN 9204
+END );
+
+ALTER TABLE qwat_od.leak DROP COLUMN type;
```

Comment:
We can't drop a column that is not it the master datamodel.

https://github.com/qwat/qwat-data-model/blob/master/ordinary_data/leak/od_leak.sql

```suggestion

```

Otherwise looks good to me

Reply:
(This first part has to be run on your client database only ;) )

Reply:
And delta should be renamed 

update/delta/delta_1.3.5_001_create_fk_leak_type.sql